catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/pom.xml

```diff
@@ -0,0 +1,207 @@
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package/>
+                        <Embed-Dependency>spatial-wfs-common;scope=compile|runtime,
+                            spatial-wfs-v1_1_0-common;scope=compile|runtime,
+                            spatial-wfs-converter;scope=compile|runtime,
+                            spatial-ogc-common;scope=compile|runtime,
+                            <!--spatial-wfs-v1_1_0-schema-bindings;scope=compile|runtime,-->
+                            catalog-core-api-impl,
+                            wfs-v_1_1_0,
+                            filter-v_1_1_0,
+                            gml-v_1_1_0
+                        </Embed-Dependency>
+                        <Embed-Transitive>true</Embed-Transitive>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>true</haltOnFailure>
+                            <rules>
+                                <rule>
+                                    <element>BUNDLE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>INSTRUCTION</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.0</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>BRANCH</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.0</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>COMPLEXITY</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.0</minimum>
+                                        </limit>
```

Comment:
Nit: extra newline

Reply:
fixed. I also just realized the coverage numbers where left at 0. I'll fix that as well.